decompiler/config/all-types.gc

```diff
@@ -1611,6 +1611,34 @@
   (hint-511                     #x14ff)
   (speedrunner-mode             #x1500)
   (cutscene-skips               #x1501)
+  (checkpoint-select            #x1502)
+  (speedrun-options-warning     #x1503)
+  (speedrun-reset-current-run   #x1504)
+  (speedrun-new-full-game       #x1505)
+  ;; (speedrun-full-game-any       #x1506)
+  ;; (speedrun-full-game-100       #x1507)
+  ;; (speedrun-full-game-nolts     #x1508)
+  ;; (speedrun-full-game-nofcs     #x1509)
+  ;; (speedrun-full-game-orbless   #x150a)
+  ;; (speedrun-full-game-all-flies #x150b)
+  ;; (speedrun-full-game-all-orbs  #x150c)
```

Comment:
Yeah just reserving space in case we ever decide to add separate menu options for these in the future for some reason, then they'll be in roughly the right spot and my OCD will be ok :)